### PR TITLE
babel-types: update validators

### PR DESCRIPTION
**Pull request by hzoo**
https://github.com/babel/babel/blob/master/doc/ast/spec.md#exports

### CODE REVIEW

File: packages/babel-types/src/definitions/es2015.js [hunk 1]
```diff
@@ -120,7 +122,9 @@ defineType("ExportDefaultDeclaration", {
   visitor: ["declaration"],
   aliases: ["Statement", "Declaration", "ModuleDeclaration", "ExportDeclaration"],
   fields: {
-    // todo
+    declaration: {
+      validate: assertNodeType("Declaration", "Expression")
```
Comment (hzoo) [inline packages/babel-types/src/definitions/es2015.js]: Although when using `Declaration` - you aren't allowed to have a `VariableDeclaration` here so maybe we should put `FunctionDeclaration` and `ClassDeclaration` instead

Comment (sebmck) [inline packages/babel-types/src/definitions/es2015.js]: Yeah, being really precise here is preferable.

**Comment by hzoo** [comment]
Not sure if I should imports the constants or just add them in the file

**Comment by sebmck** [comment]
Thanks @hzoo!

**Pull request merged**